pages/api/users/[userId].ts

```diff
@@ -0,0 +1,23 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma from "../../../lib/prisma";
+
+export default async function handle(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { userId: userIdParam } = req.query;
+  if (userIdParam instanceof Array) {
```

Comment:
Out of curiosity, was it parsing as an array somehow? I would have guessed they had to be strings

Reply:
The type in typescript was string or array of strings, not sure why

Reply:
I think this is because it can include params passed at the end of the uri via foo=bar and there can be multiple